app/views/users/_profile.html.erb

```diff
@@ -45,7 +45,10 @@
       </span>
     <% end %>
   </div>
-
+    <div class="field checkbox-label-first">
+    <%= f.label :allows_article_edits, "Allow admins to edit typos in my articles" %>
```

Comment:
I think this should be "Allow admins to edit typos or adjust formatting in my articles." Wanted to add the formatting part because sometimes we adjust markdown errors.

Reply:
totally -- will add